gulpfile.js/tasks/copy.js

```diff
@@ -3,11 +3,11 @@ const gulp = require('gulp')
 module.exports = function (config) {
   gulp.task('copy', function () {
     const changed = require('gulp-changed')
-    const handleErrors = require('../utils/handleErrors')
+    // const handleErrors = require('../utils/handleErrors')
     let task = gulp.src(config.copy)
-    .pipe(changed(config.dest))
-    .pipe(gulp.dest(config.dest))
-    .on('error', handleErrors)
+      .pipe(changed(config.dest))
+      .pipe(gulp.dest(config.dest))
+    // .on('error', handleErrors)
```

Comment:
remove?

Reply:
will be removed before merge in master. still doing some testing